lib/code_corps_web/helpers/authentication_helpers.ex

```diff
@@ -1,4 +1,7 @@
 defmodule CodeCorpsWeb.AuthenticationHelpers do
+  @moduledoc """
+  AuthenticationHelpers
+  """
```

Comment:
Thinking about this some more, I think I'd rather we keep the warning up.

Sure, this removes a credo warning, but all that does for us is to hide the fact that we don't have this module properly documented. 

@JoshSmith How do you feel about it?

My vote is, remove these cases from the PR. If credo complains, let it complain until we properly fix it.

Reply:
Yeah, I started adding in a few "empty" moduledocs, and then stopped when I realized it would be like ~200 different silly changes, and I disabled the check altogether.

Let me know what the plan is. My suggestion: I can remove these empty moduledocs, and re-enable the check. 👍

Reply:
@untra Yeah, I'm thinking for now, go with that. We want our code properly documented, and these "empty" moduledocs would just hide the fact that some modules are not documented.

Reply:
Seems you'll also have some conflicts to resolve in `policy.ex`. Sadly, not much to do to get around that.

Reply:
I agree here.